### PR TITLE
Inheritance

### PR DESCRIPTION
**Pull request by elizaochoa86**
Added new classes of inheritance's projects

### CODE REVIEW

File: src/main/java/com/learn/java/inheritance/AbstractFlyer.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.learn.java.inheritance;
+
+public abstract class AbstractFlyer implements Flyer {
+    public void fly(){
+        System.out.println(1);
```
Comment (reyniergt) [inline src/main/java/com/learn/java/inheritance/AbstractFlyer.java]: modify the fly method in order to use a protected field  _flyer_ and print "the _flyer_ flies"
the flyer field should not allow to be modified

File: src/main/java/com/learn/java/inheritance/AbstractRunner.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.learn.java.inheritance;
+
+public abstract class AbstractRunner implements Runner {
+    public void run(){
+        System.out.println(2);
```
Comment (reyniergt) [inline src/main/java/com/learn/java/inheritance/AbstractRunner.java]: modify the run method in order to use a protected field _runner_ and print "the _runner_ runs"

File: src/main/java/com/learn/java/inheritance/AbstractSwimmer.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package com.learn.java.inheritance;
+
+public abstract class AbstractSwimmer implements Swimmer {
+    public void swim(){
+        System.out.println(3);
```
Comment (reyniergt) [inline src/main/java/com/learn/java/inheritance/AbstractSwimmer.java]: modify the swim method in order to use a protected field  _swimmer_ and print "the _swimmer_ swims"

File: src/main/java/com/learn/java/inheritance/Cat.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package com.learn.java.inheritance;
+
+public class Cat extends AbstractRunner {
+
```
Comment (reyniergt) [inline src/main/java/com/learn/java/inheritance/Cat.java:4]: assign the value Cat to the _runner_ field

File: src/main/java/com/learn/java/inheritance/Dog.java [hunk 5]
```diff
@@ -0,0 +1,5 @@
+package com.learn.java.inheritance;
+
+public class Dog extends AbstractRunner {
+
```
Comment (reyniergt) [inline src/main/java/com/learn/java/inheritance/Dog.java:4]: assign the value Dog to the runner field

File: src/main/java/com/learn/java/inheritance/Dolphin.java [hunk 6]
```diff
@@ -0,0 +1,5 @@
+package com.learn.java.inheritance;
+
+public class Dolphin extends AbstractSwimmer {
+
```
Comment (reyniergt) [inline src/main/java/com/learn/java/inheritance/Dolphin.java]: assign the value Dolphin to the _swimmer_ field
Do the same for the rest of the classes

**Review by reyniergt — CHANGES_REQUESTED** [review]
verify comments

**Comment by elizaochoa86** [comment]
Changes are reading!

**Review by reyniergt — APPROVED** [review]
**Great work!!**

**Pull request merged**